spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
Well, validation itself takes up some time, as well, but basically each style is read and parsed (by nokogiri) twice. We'd have to make some changes to CSL-Ruby to allow us to feed nokogiri's parse results directly into the validator and style parser, then we could essentially run both in one go.